.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*.{kt,kts}]
+
```

Comment:
Need to declare the style explicitly in order to make rules specified effective.

```kotlin
ktlint_code_style = ktlint_official
```